### ISSUE TITLE
Request: Bag opening change [1.12.2]

### ISSUE THREAD

**Issue by fkrisi11** [MrZarak/MultiBags#13]
Could you add a little icon in the vanilla inventory screen, to the top left of the chracter portrait to open the bag?

It would be much nicer to have that little button there, than to open the bag ui instantly with the E key.
Maybe make it a config option:
"Open equipped bags with the inventory key: true/false"

The small icon I was talking about could be there regardless.

**Comment by fkrisi11** [comment]
Maybe the little icon I was talking about could just show or hide the ui of the bag.
But when you press E, the bag shouldn't open right away in any case.

**Comment by ghost** [comment]
I completely agree with this. It would be much more convenient to open a bag from regular inventory like baubles menu.
Of course, it should automatically close when player closes his inventory via his inventory key or Esc, so when he opens his inventory again - regular inventory should open, not bag inventory.
